src/screens/certificate/CertificateIncome.tsx

```diff
@@ -0,0 +1,128 @@
+import React, { useEffect, useState } from 'react';
+import { StyleSheet, View, ViewStyle } from 'react-native';
+
+import { Button } from '../../components/Button';
+import Card from '../../components/Card';
+import Screen from '../../components/Screen';
+import Text from '../../components/Text';
+import { useClient } from '../../data/client';
+import { useAppSelector, useGlobalStyles } from '../../hooks';
+import { RequestType } from '../../models/results';
+import { getStudentYear } from '../../utils/datetime';
+import composeMail from '../../utils/email';
+import { fontSize } from '../../utils/texts';
+import { Input, PopoverElement } from './CertificateComponents';
+
+const stipEmail = 'stip@psu.ru';
+
+const makeMailOptions = ({
+  fio,
+  faculty,
+  year,
+  certPeriod,
+}: {
+  fio: string;
+  faculty: string;
+  year: string;
+  certPeriod: string;
+}) => {
+  return {
+    recipients: [stipEmail],
+    subject: 'Справка о доходах',
+    body: `1. ФИО: ${fio}\n2. Факультет: ${faculty}, курс: ${year}\n3. Период в месяцах: ${certPeriod}`,
+  };
+};
+
+const btnCompose: ViewStyle = { position: 'absolute', left: 0, right: 0, bottom: '1%' };
+
+const styles = StyleSheet.create({
+  btnCompose: btnCompose,
+  btnComposeDisabled: {
+    ...btnCompose,
+    opacity: 0.75,
+  },
+});
+
+export default function CertificateIncome() {
+  const globalStyles = useGlobalStyles();
+  const client = useClient();
+
+  const { info } = useAppSelector((state) => state.student);
+  const [fio, setFio] = useState<string>(info.name);
+  const [faculty, setFaculty] = useState<string>();
+  const [year, setYear] = useState<string>(getStudentYear(Number.parseInt(info.year)));
+  const [certPeriod, setCertPeriod] = useState<string>();
+
+  const applicable: boolean = !!fio && !!faculty && !!year && !!certPeriod;
+
+  useEffect(() => {
+    const fetchFaculty = async () => {
+      try {
+        const data = await client.getPersonalRecords({ requestType: RequestType.tryFetch });
```

Comment:
```suggestion
        const response = await client.getPersonalRecords({ requestType: RequestType.tryCache});
```
Каждый раз, при запуске приложения происходит фетч личных записей, поэтому не вижу смысла ещё раз фетчить здесь